data-migration/aurora/from-snapshot.md

```diff
@@ -5,120 +5,114 @@ summary: Learn how to migrate full data from Amazon Aurora MySQL to TiDB using T
 
 # Migrate from Aurora snapshot to TiDB
 
-This document introduces how to migrate full data from Amazon Aurora MySQL to TiDB using TiDB Lightning.
+This document describes how to migrate full data from Amazon Aurora MySQL to TiDB using TiDB Lightning. Both Dumpling and TiDB Lighting are used for this migration.
 
-TiDB Lightning is a tool used for fast full import of large amounts of data into a TiDB cluster.
+## Prerequisites
 
-Currently, TiDB Lightning can mainly be used in the following two scenarios:
+- [Deploy TiDB Lighting using TiUP](/data-migration/quick_install_tools.md)
+- [Deploy Dumping using TiUP](/data-migration/quick_install_tools.md)
 
-- Importing large amounts of new data quickly
-- Restore all backup data
+***
 
-Currently, TiDB Lightning supports:
+## Step 1. Export snapshot data of Aurora to Amazon S3
 
-- The data source of the Dumpling, CSV or Amazon Aurora Parquet exported formats.
-- Reading data from a local disk or from the Amazon S3 storage. For details, see [External Storages](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages).
+To export a snapshot to Amazon S3, refer to Amazon's official document [Exporting DB snapshot data to Amazon S3](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_ExportSnapshot.html).
 
-## Before starting the migration, complete the following tasks
+## Step 2. Export table schema files using Dumpling 
 
-Because the snapshot data exported from Aurora to S3 does not contain the SQL statement file used to create database tables, you need to manually export and import the table creation statements corresponding to the database tables into TiDB. You can use Dumpling and TiDB Lightning to create all table schemas:
+Because the snapshot data exported to Amazon S3 does not contain the SQL statement file for creating database tables, you need to manually export and import the table creation statements corresponding to the database tables into TiDB. You can use Dumpling and TiDB Lightning to create all table schemas:
```

Comment:
```suggestion
Because the snapshot data exported to Amazon S3 does not contain the SQL statement file for creating database tables, you need to manually export and import the table creation statements into TiDB. You can use Dumpling and TiDB Lightning to create all table schemas:
```